packages/editor-ui/src/components/NodeIcon.vue

```diff
@@ -1,34 +1,38 @@
 <template>
-	<div class="node-icon-wrapper" :style="iconStyleData">
-		<div v-if="nodeIconData !== null" class="icon">
-			<img v-if="nodeIconData.type === 'file'" :src="nodeIconData.fileBuffer || nodeIconData.path" :style="imageStyleData" />
-			<font-awesome-icon v-else :icon="nodeIconData.icon || nodeIconData.path" :style="fontStyleData" />
-		</div>
-		<div v-else class="node-icon-placeholder">
-			{{nodeType !== null ? nodeType.displayName.charAt(0) : '?' }}
-		</div>
-	</div>
+	<n8n-node-icon
+		:type="nodeIconData.type"
+		:path="nodeIconData.path || nodeIconData.icon || nodeIconData.fileBuffer"
+		:color="nodeIconData.color"
+		:disabled="disabled"
+		:size="size"
+		:circle="circle"
+		:nodeTypeName="nodeType.displayName"
+		:showTooltip="showTooltip"
+	></n8n-node-icon>
 </template>
 
 <script lang="ts">
-
 import { IVersionNode } from '@/Interface';
 import { INodeTypeDescription } from 'n8n-workflow';
+import N8nNodeIcon from '../../../design-system/src/components/N8nNodeIcon/NodeIcon.vue';
```

Comment:
This import is not needed since you're declaring the component globally in `plugins/components.ts`.

Reply:
Ah yeah. I was using it for testing and forgot to remove. Thanks.